src/java.base/share/classes/java/lang/invoke/MethodHandleImpl.java

```diff
@@ -25,15 +25,14 @@
 
 /*
  * ===========================================================================
- * (c) Copyright IBM Corp. 2022, 2022 All Rights Reserved
+ * (c) Copyright IBM Corp. 2022, 2024 All Rights Reserved
  * ===========================================================================
  */
 
 package java.lang.invoke;
 
 import jdk.internal.access.JavaLangInvokeAccess;
 import jdk.internal.access.SharedSecrets;
-import jdk.internal.foreign.abi.NativeEntryPoint;
```

Comment:
I think you should leave this alone; it isn't used, but there's no harm keeping it.

Reply:
Fixed.